### PR TITLE
magento-engcom/community-portal#8 Configure serverless to create DynamoDB tables

### PR DESCRIPTION
**Pull request by XiyaMagento**
magento-engcom/community-portal#8 Configure serverless to create DynamoDB tables

- configure connections to DynamoDB
- write READ and CREATE handlers in TypeScript
- write tests to check READ and CREATE operations on local and remote DynamoDB

### CODE REVIEW

File: functions/__tests__/dynomodb_local_test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const rp = require('request-promise');
```
Comment (vrann) [inline functions/__tests__/dynomodb_local_test.ts]: let's keep `tests` directory with no underscores for the tests

Comment (vrann) [inline functions/__tests__/dynomodb_local_test.ts]: also, `axios` https://github.com/axios/axios library seems much more widely used for this type of functionality. Any reasons to prefer request-promise over it?

File: functions/__tests__/dynomodb_local_test.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+const postOptions = {
+  method: 'POST',
+  uri: 'http://localhost:3000/create',
```
[the remark this answers is not in the document]
Comment (XiyaMagento) [reply functions/__tests__/dynomodb_local_test.ts]: I'll check and let you know

File: functions/__tests__/dynomodb_local_test.ts (hunk 1 above)
Comment (XiyaMagento) [reply functions/__tests__/dynomodb_local_test.ts]: Got it

Comment (XiyaMagento) [reply functions/__tests__/dynomodb_local_test.ts]: I see, I'll change to axios instead

**Comment by XiyaMagento** [comment]
Epic 1 completed. Closing this PR.

**Pull request closed without merging**